redirects.next.js

```diff
@@ -192,6 +192,32 @@ module.exports = (async () => {
     '/cloud-docs/api-docs/providers': '/cloud-docs/api-docs/private-registry/providers',
     '/cloud-docs/api-docs/modules': '/cloud-docs/api-docs/private-registry/modules',
+    '/enterprise/install': '/enterprise/install/pre-install-checklist',
+    '/enterprise/before-installing': '/enterprise/install/pre-install-checklist',
+    '/enterprise/before-installing/reference-architecture': '/enterprise/reference-architecture',
+    '/enterprise/before-installing/reference-architecture/aws': '/enterprise/reference-architecture/aws',
+    '/enterprise/before-installing/reference-architecture/azure': '/enterprise/reference-architecture/azure',
+    '/enterprise/before-installing/reference-architecture/gcp': '/enterprise/reference-architecture/gcp',
+    '/enterprise/before-installing/reference-architecture/vmware': '/enterprise/reference-architecture/vmware',
+    '/enterprise/before-installing/os-specific/rhel-requirements': '/enterprise/requirements/os-specific/rhel-requirements',
+    '/enterprise/before-installing/os-specific/centos-requirements': '/enterprise/requirements/os-specific/centos-requirements',
+    '/enterprise/before-installing/network-requirements': '/enterprise/requirements/network',
+    '/enterprise/user-management/saml': '/enterprise/saml/configuration',
+    '/enterprise/user-management/saml/configuration': '/enterprise/saml/configuration',
+    '/enterprise/user-management/saml/team-membership': '/enterprise/saml/team-membership',
+    '/enterprise/user-management/saml/attributes': '/enterprise/saml/attributes',
+    '/enterprise/user-management/saml/login': '/enterprise/saml/login',
+    '/enterprise/user-management/saml/troubleshooting': '/enterprise/saml/troubleshooting',
+    '/enterprise/user-management/saml/idp-configuration': '/enterprise/saml/idp-configuration',
+    '/enterprise/user-management/saml/idp-configuration/adfs': '/enterprise/saml/idp-configuration/adfs',
+    '/enterprise/user-management/saml/idp-configuration/aad': '/enterprise/saml/idp-configuration/aad',
+    '/enterprise/user-management/saml/idp-configuration/okta': '/enterprise/saml/idp-configuration/okta',
+    '/enterprise/user-management/saml/idp-configuration/onelogin': '/enterprise/saml/idp-configuration/onelogin',
+    '/enterprise/before-installing/data-storage-requirements/postgres-requirements': '/enterprise/requirements/data-storage/postgres-requirements',
+    '/enterprise/before-installing/data-storage-requirements/disk-requirements': '/enterprise/requirements/data-storage/disk-requirements',
+    '/enterprise/before-installing/data-storage-requirements/minio-setup-guide': '/enterprise/requirements/data-storage/mineo-setup-guide',
+    '/enterprise/before-installing/data-storage-requirements/vault': '/enterprise/requirements/data-storage/vault',
+    '/enterprise/requirements/data-storage/disk-requirements': '/enterprise/requirements/data-storage/operational-mode-requirements',
```

Comment:
This removes 1 _redirect-to-a-redirect_

```suggestion
    '/enterprise/before-installing/data-storage-requirements/postgres-requirements': '/enterprise/requirements/data-storage/postgres-requirements',
    '/enterprise/before-installing/data-storage-requirements/disk-requirements': '/enterprise/requirements/data-storage/operational-mode-requirements',
    '/enterprise/before-installing/data-storage-requirements/minio-setup-guide': '/enterprise/requirements/data-storage/mineo-setup-guide',
    '/enterprise/before-installing/data-storage-requirements/vault': '/enterprise/requirements/data-storage/vault',
```